hangups_client.py

```diff
@@ -207,6 +219,13 @@ def _on_message_sent(future):
     # that the bot read the message)
     #print_jsonmsg(json.dumps({"status":"success"}))
 
+async def download_image(url):
```

Comment:
sorry, this code will break < python 3.5
but I don't know old version `async with` replacement.